config/routes/liip_imagine.yaml

```
@@ -0,0 +1,2 @@
+_liip_imagine:
```

Why the leading underscore?

```suggestion
liip_imagine:
```

---

Don't think we should change, it' auto-generated by the bundle and works.